types/defines/event-subscriptions.d.ts

```diff
@@ -0,0 +1,80 @@
+type EventSubscriptionEvent<
+	Service extends string,
+	Type extends string,
+	Source,
+	Payload = {},
```

Comment:
I think this should probably default to `Record<string, never>` instead? (assuming this is meant to be an empty object)